app/assets/javascripts/components/containers/mastodon.jsx

```diff
@@ -89,6 +90,7 @@ addLocaleData([
   ...uk,
   ...zh,
```

Comment:
I mean, it's like nobody is gonna make a `zh` locale and have a label for that...